### ISSUE TITLE
bug: accessibility issues in whale bird.

### ISSUE THREAD

**Issue by marrie** [h3poteto/whalebird-desktop#3800]
# issues

there are some accessibility issues in whale bird such as not naming the account I’m focused on with control and account keys and in the system tray icon, not speaking the words and or chars my curser over when I’m typing to correct my work and unlabeled menus in the side bar

# use case

I use nvda, a screen reader which can be found at the [following website.](http://nvda-project.org) When I’m using the app, I cannot see the account I’m currently on as all it lists are the names of the instances, I’m on. This is of no help as two of my accounts are on the same instance.

Second, the menus are not labeled in the side bar, hence I have to guess what menu is what. This prevents me from taking full advantage of some features I might be missing out on as the menus are not labeled. All I hear is “menu collapsed” or “menu expanded” if I hit enter on said menu.

Third: I cannot see what I’m typing as text does not read when I’m arrowing through the text. I must go into WinWord or notepad to type what I want, then paste my work into the app. This is a work around for now, although I wish this would be fixed so I don’t always have WinWord open to type out my thought.

Can these issues be looked at? I love the app over all and love the way it works; it is so lite on my system I barely feel it. I’d love to continue using this app, if the accessibility can be made much better, the visually impaired community would really appreciate this.

Thanks.
